_posts/2019-06-10-folsom-where-we-work-and-play.md

```diff
@@ -1,12 +1,13 @@
 ---
```

Comment:
This is still a good post, but it's misleading now that we've moved. Should we update the post with a note that we've moved?